imports/plugins/core/system-info/server/no-meteor/resolvers/Query/systemInformation.js

```diff
@@ -0,0 +1,15 @@
+import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+
+/**
+ * @name Query.systemInformation
+ * @method
+ * @memberof SystemInformation/GraphQL
+ * @summary get system information for reaction site
+ * @param {ConnectionArgs} args An object of all arguments that were sent by the client
+ * @param {Object} context An object containing the per-request state
+ * @return {<Object>} System Information Object
```

Comment:
In this function and the other query function, the `@return` type is technically `Promise<Object>` since they are `async`.